docs/UserGuide.adoc

```diff
@@ -162,25 +162,33 @@ recipe to contain two instructions:
     * boil eggs
     * slice apples
 
-=== List Recipes — `list`
-Lists out all the recipes managed by *CookBuddy*.
+=== Recipe Listing — `list`
+You can list all the existing recipes from *CookBuddy* using the `list` command.
 
-=== Find Recipes — `find`
-Find recipes with a particular parameter.
+Format: `list`
+
+=== Recipe Finding — `find`
```

Comment:
```suggestion
=== Finding recipes — `find`
```
let's keep the naming conventions consistent for now. If we agree to phrase it differently later, then we'll open a new PR.